pkg/storer/internal/reserve/reserve_test.go

```diff
@@ -218,17 +218,26 @@ func TestEvict(t *testing.T) {
 		}
 	}
 
-	totalEvicted := 0
+	chsToDelete := make([]swarm.Chunk, 0)
```

Comment:
The slice length is missing.

Reply:
This is not required right? There is a check based on length, so I dont want to preallocate.